server/src/infra/repositories/machine-learning.repository.ts

```diff
@@ -1,21 +1,44 @@
 import { DetectFaceResult, IMachineLearningRepository, MachineLearningInput, MACHINE_LEARNING_URL } from '@app/domain';
 import { Injectable } from '@nestjs/common';
 import axios from 'axios';
+import FormData from 'form-data';
+import { createReadStream } from 'fs';
 
 const client = axios.create({ baseURL: MACHINE_LEARNING_URL });
 
 @Injectable()
 export class MachineLearningRepository implements IMachineLearningRepository {
   classifyImage(input: MachineLearningInput): Promise<string[]> {
-    return client.post<string[]>('/image-classifier/tag-image', input).then((res) => res.data);
+    const formData = new FormData();
+    const fileStream = createReadStream(input.imagePath);
+    formData.append('image', fileStream);
+    return client
+      .post<string[]>('/image-classifier/tag-image', formData, {
+        headers: formData.getHeaders(),
+      })
+      .then((res) => res.data);
```

Comment:
Can you extract this duplicate code into a separate function that you use for all the requests?

Reply:
fixed this